Source/Core/Core.swift

```diff
@@ -912,7 +912,7 @@ extension FormViewController : UITableViewDelegate {
 			}
             return .none
         }
-        if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1 {
+        if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1 && section.showInsertIconInAddButton {
             return .insert
         }
```

Comment:
I think the proper code is: 
```swift
 if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1{ 
      return section.showInsertIconInAddButton ? .insert : .none
}
```

Reply:
It will fall through so my code is also proper one. I don't see any advantages compared to your code.

Reply:
For the last row if showInsertIconInAddButton is false and multivalued section allows Insertion and deletion your code display a delete icon when it should display nothing. 

Hope this clarify something.

Reply:
Yes, now I understand. Let me change this.